client/interfaces.go

```diff
@@ -136,6 +136,63 @@ type InstanceServer interface {
 	UpdateContainerTemplateFile(containerName string, templateName string, content io.ReadSeeker) (err error)
 	DeleteContainerTemplateFile(name string, templateName string) (err error)
 
+	// Instance functions.
+	GetInstanceNames(instanceType api.InstanceType) (names []string, err error)
+	GetInstances(instanceType api.InstanceType) (instances []api.Instance, err error)
+	GetInstancesFull(instanceType api.InstanceType) (instances []api.InstanceFull, err error)
+	GetInstance(name string) (instance *api.Instance, ETag string, err error)
+	CreateInstance(instance api.InstancesPost) (op Operation, err error)
+	CreateInstanceFromImage(source ImageServer, image api.Image, imgcontainer api.InstancesPost) (op RemoteOperation, err error)
```

Comment:
Should change `imgContainer` there for `req` I think

Reply:
@stgraber will do

Reply:
@stgraber done